### DISCUSSION TITLE
Having issues where ocassionally the Semantic Kernel doesn't go through one known workflow path / plugin.

### DISCUSSION THREAD

**Discussion by capgemhon** [microsoft/semantic-kernel#6294] [General]
Hi there,

I am very occasionally get the Semantic Kernel responding with "I apologize for the inconvenience, but it seems there is a technical issue preventing me from accessing the '-----.pdf' document. Therefore, I am unable to provide a summary at this time, after a  document upload. It can sometimes repeat that answer and I have seen it not gone through any of the plugins, and suspect its gone through the default SDK workflow/pathway or some other pathway. Is there a known way or ways to reduce this problem occurring? Thanks, Hon

**Comment by madsbolaris** [comment]
Are you able to share what plugins you're giving to the LLM? It _sounds_ like it's trying to use one of the plugins, but is receiving some sort of error. If you're able to provide more information, we may be able to help.
